L03-generics/src/main/java/ru/otus/homework/CustomerReverseOrder.java

```diff
@@ -0,0 +1,16 @@
+package ru.otus.homework;
+
+import java.util.LinkedList;
+
+public class CustomerReverseOrder {
+
+    private final LinkedList<Customer> customers = new LinkedList<>();
```

Comment:
Тип лучше объявлять интерфейсом.

```suggestion
    private final Deque<Customer> customers = new LinkedList<>();
```